saffron/src/blob.rs

```diff
@@ -110,11 +114,51 @@ impl<G: KimchiCurve> FieldBlob<G> {
         bytes.truncate(blob.n_bytes);
         bytes
     }
+
+    #[instrument(skip_all, level = "debug")]
+    pub fn update<EFqSponge: Clone + FqSponge<G::BaseField, G, G::ScalarField>>(
+        &mut self,
+        srs: &SRS<G>,
+        domain: &Radix2EvaluationDomain<G::ScalarField>,
+        diff: Diff<G::ScalarField>,
+    ) {
+        let updates: Vec<(usize, PolyComm<G>, DensePolynomial<G::ScalarField>)> = diff
```

Comment:
That's not what we want to do, it is not efficient. You only want to use the Lagrange polynomials.

Reply:
Yeah this is what I figured as well, but I saw while working on this that it isn't part of the `SRS` (just the commitments). I didn't see another type in the codebase that manages this cache, am i missing something? 

If we need to make something, this updat needs the lagrange polynomials in the monomial basis since this is what the data is stored in. If it's an expensive one time set up (seems like it to me), can I suggest leaving a TODO and a follow up PR which:

- creates a type that manages the cache of lagrange polynomials for a given domain
- provied (de)serialize instances
- add a cli arg to load the cache from a file, apply it where necessary 
- save the cache in CI.

Reply:
We don't want to store the polynomials in the monomial basis, this has to be fixed as said in DM. You keep the evaluations. From there, you simply commit using the SRS on the diff.